docs/guides/guide-vmanomaly-vmalert.md

```diff
@@ -0,0 +1,403 @@
+# VmAnomaly Quickstart
+
+### Prerequisites
+- In this tutorial it is assumed that you have VictoriaMetrics already, and you are familiar with VictoriaMetrics ecosystem:
+  -  [victoriametrics](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html)
+  -  [vmalert](https://docs.victoriametrics.com/vmalert.html)
+  -  [vmagent](https://docs.victoriametrics.com/vmagent.html)
+  
+- It is assumed that you are familiar with Grafana and Docker
+## What is VmAnomaly?
+*VictoriaMetrics Anomaly Detection* is a service that continuously scans VictoriaMetrics time series and detects unexpected changes within data patterns in real-time. It does so by utilizing user-configurable machine learning models.
```

Comment:
```suggestion
*VictoriaMetrics Anomaly Detection* is a service that continuously scans time series stored in VictoriaMetrics and detects unexpected changes within data patterns in real-time. It does so by utilizing user-configurable machine learning models.
```

Reply:
> machine learning models

Do we have a list of supported models? If yes, can we link them here?

Reply:
changed the sentence.
We have brief list on vmanomaly doc page, but there are no specifications given.
So the full list is yet to come and I will add it when it is ready.